content/en/docs/setup/getting-started/index.md

```diff
@@ -353,7 +353,9 @@ Use the following instructions to deploy the [Kiali](/docs/ops/integrations/kial
 
     {{< text bash >}}
     $ kubectl apply -f @samples/addons@
-    $ while ! kubectl wait --for=condition=available --timeout=600s deployment/kiali -n istio-system; do sleep 1; done
+    $ kubectl rollout status deployment/kiali -n istio-system
+    Waiting for deployment "kiali" rollout to finish: 0 of 1 updated replicas are available...
+    deployment "kiali" successfully rolled out
```

Comment:
I think this warrants a change in getting-started/test.sh.
https://github.com/istio/istio.io/blob/04a158603c8d55df0db926a8a5e67dbffed1531c/content/en/docs/setup/getting-started/test.sh#L80
Change line 80 to something like this:
```
_verify_contains snip_view_the_dashboard_dashboard_1 'deployment "kiali" successfully rolled out'
```

Reply:
Done